src/bitmessagecli.py

```diff
@@ -23,32 +23,36 @@
 api = ''
 keysName = 'keys.dat'
 keysPath = 'keys.dat'
-usrPrompt = 0 #0 = First Start, 1 = prompt, 2 = no prompt if the program is starting up
+usrPrompt = 0  # 0 = First Start, 1 = prompt, 2 = no prompt if the program is starting up
 knownAddresses = dict()
 
-def userInput(message): #Checks input for exit or quit. Also formats for input, etc
+
+def userInput(message):  # Checks input for exit or quit. Also formats for input, etc
     global usrPrompt
     print '\n' + message
     uInput = raw_input('> ')
 
-    if (uInput.lower() == 'exit'): #Returns the user to the main menu
+    if (uInput.lower() == 'exit'):  # Returns the user to the main menu
```

Comment:
Brackets are redundant here

Reply:
There are a bunch of these. They are highlighted by the tool. The only changes here have been to run autopep8. Once as many errors and warnings are being fixed by the tool I will do the manual fixes. Not sure whether I'll find a tool to auto-fix superfluous-parens warning but it will get done, thanks for pointing it out.